agent/conf/log4j-cloud.xml.in

```diff
@@ -29,10 +29,15 @@ under the License.
    <appender name="FILE" class="org.apache.log4j.rolling.RollingFileAppender">
       <param name="Append" value="true"/>
       <param name="Threshold" value="INFO"/>
-      <rollingPolicy class="org.apache.log4j.rolling.TimeBasedRollingPolicy">
-        <param name="FileNamePattern" value="@AGENTLOG@.%d{yyyy-MM-dd}.gz"/>
+      <rollingPolicy class="org.apache.log4j.rolling.FixedWindowRollingPolicy">
+        <param name="FileNamePattern" value="@AGENTLOG@.%i.gz"/>
```

Comment:
This may cause some admins stress, given many may be used to date/time stamp based logs. If the fix is limited to the log4j xml configs, you can simply configure this for your environments (not necessarily upstream codebase).

Reply:
Or, discuss this on dev@ and users@ gather consensus and go forward with this chage - https://cloudstack.apache.org/mailing-lists.html